src/analysis/scripts/export_connectivity.sh

```diff
@@ -75,6 +75,35 @@ function ec_export_table_csv() {
          -c "\COPY ${EXPORT_TABLENAME} TO '${FILENAME}' WITH (FORMAT CSV, HEADER)"
 }
 
+function ec_export_table_geojson() {
+  OUTPUT_DIR="${1}"
+  EXPORT_TABLENAME="${2}"
+
+  FILENAME="${OUTPUT_DIR}/${EXPORT_TABLENAME}.geojson"
+  ogr2ogr -f GeoJSON "${FILENAME}" \
+          -t_srs EPSG:4326 \
```

Comment:
Setting `-s_srs` doesn't seem to be necessary for now. I pushed a commit to remove it. It can be added back in later if it causes trouble for other neighborhoods.